src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line at line 8 should be removed.